projects/dsp-ui/package.json

```diff
@@ -1,15 +1,21 @@
 {
   "name": "@dasch-swiss/dsp-ui",
-  "version": "1.0.0-rc.1",
+  "version": "1.0.0-rc.2",
   "description": "Angular library to easily create a Knora/DSP user interface",
   "repository": {
     "type": "git",
     "url": "https://github.com/dasch-swiss/dsp-ui-lib.git"
   },
   "homepage": "https://docs.dasch.swiss/developers/dsp-ui/documentation/",
-  "keywords": ["dasch service platform", "user interface", "angular", "knora", "humanities"],
+  "keywords": [
+    "dasch service platform",
+    "user interface",
+    "angular",
+    "knora",
+    "humanities"
+  ],
   "author": "Data and Service Center for the Humanities DaSCH, University of Basel",
-  "license" : "AGPL-3.0-or-later",
+  "license": "AGPL-3.0-or-later",
```

Comment:
This looks great. Can you do the same in main `package.json`?

Reply:
what is different in `package.json`? I cannot see any

Reply:
ah keywords maybe?

Reply:
This was nonsense. I'm sorry